staging_service/import_specifications/file_parser.py

```diff
@@ -8,7 +8,7 @@
 # TODO update to C impl when fixed: https://github.com/Marco-Sulla/python-frozendict/issues/26
 from frozendict.core import frozendict
 from pathlib import Path
-from typing import Tuple, Union
+from typing import Union, Callable
```

Comment:
Callable is now generic if you do `from __future__ import annotations` as of 3.9 as per https://www.python.org/dev/peps/pep-0585/

Reply:
I think that's something different, per the PEP it's this: https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable

Reply:
Whereas a typing.Callable represents a function

Reply:
https://docs.python.org/3/library/typing.html#callable

Reply:
Yea you are right